### ISSUE TITLE
Precision Improvement of BFCL on the Released Dataset After Training

### ISSUE THREAD

**Issue by LetsGoFir** [inclusionAI/AWorld-RL#8]
Hi! Thanks for your great job. I conducted separate SFT and RL training on the released HardGen dataset, but the model precision shows no improvement after both training phases. I would appreciate it if you could help me troubleshooting the possible causes.

Key Training Configs (8 GPUs for both SFT/RL)

- SFT Training Params
--num_train_epochs 5
--learning_rate 4e-5
--global_batchsize 128

- RL Training Params (GRPO-based)
BATCH_SIZE=512  # Total samples per rollout = PPO_MINI_BATCH_SIZE * model sync interval
PPO_MINI_BATCH_SIZE=128  # Per-GPU mini-batch * GPU num * gradient accumulation steps
MICRO_BATCH_SIZE=4  # Per-GPU batch size (memory constraint)
ROLLOUT_GPU_MEMORY_UTILIZATION=0.5  # Max vLLM memory util for rollout
TP=2  # Tensor parallelism
lr=1e-6
epoch=4
use the final turn response to pred and calculate the reward

- Common Preprocessing Step
Converted dataset format from AST (list of commands) to JSON format for both SFT and RL training.
Remove `think` blocks

I can upload the file if you want it.

**Comment by LetsGoFir** [comment]
> I think you can upload an example of data that you converted to huggingface then I can help with you~

I got improvement by mixing other data : )
